src/index.js

```diff
@@ -44,7 +44,34 @@ window.addEventListener('load', (event) => {
   // Bind the click event of each element to a function
   document.querySelectorAll('.card').forEach((card) => {
     card.addEventListener('click', () => {
-      // TODO: write some code here
+      
+      let countPairsClicked = memoryGame.pairsClicked;
+      let countPairsGuessed = memoryGame.pairsGuessed;
+
+      if (memoryGame.pickedCards.length = 2) {
```

Comment:
```suggestion
      if (memoryGame.pickedCards.length === 2) {
```

Con un solo igual lo que estarias haciendo es asignar el valor y me figuro que lo que quieres es evaluarlo :D